.github/workflows/pr-check.yaml

```diff
@@ -35,7 +35,18 @@ jobs:
                   for i in `jq -r .extensions[].versions[].ociUri api/extensions.json`
                   do
                     echo "Verify image $i"
-                    podman pull -q $i
+                    CONTAINER_ID=$(podman create -q --entrypoint sh $i)
```

Comment:
Wondering if this would work as extension images are created from scratch ?

Reply:
yes it works

Reply:
in fact on recent podman version you don't need to provide the entrypoint on create but on Ubuntu's version if you don't you have an error saying that the entrypoint is missing ...

Reply:
note: it's a create command, not a run/start command